guides/Shopify.md

```diff
@@ -0,0 +1,53 @@
+# Setting deploys to Shopify via CircleCI
+
+## Problem:
+
+By the reason of specificity of work with Shopify via `grunt-shopify`, we uploading changed file each time we save it. This may cause situations, when we push some changes to the server, then discard them with turned off `grunt`. So, we have some changes on the server and no records in git about them.  
+
+## Solution:
+
+The solution is to synchronise deploy branches (`master` and `staging`) with servers and automatically deploy via CircleCI on each of the pushes to the related branch.
+
+### For this you will need to: 
+
+* Add the application to the CircleCI
+* Create a `circle.yml` file in the root of the application and add it to the GIT
+* Fill the `circle.yml` with deployment commands information:
+Example:
+```
+deployment:
+  production:
+    branch: master
+    commands:
+      - grunt shopify:upload
+  staging:
+    branch: staging
+    commands:
+      - grunt shopify:upload
+```
+
+* Update `gruntfile.js` file to read the Shopify credentials from the environment variables:
+Example:
+```
+if(process.env.IS_CI) {
+  if(process.env.CIRCLE_BRANCH == 'master') {
+    var environment = 'PRODUCTION';
+  } else {
+    var environment = 'STAGING';
+  }
+  var localConfig = {};
+  localConfig[':api_key'] = process.env[environment + '_SHOPIFY_API_KEY'];
+  localConfig[':password'] = process.env[environment + '_SHOPIFY_PASSWORD'];
+  localConfig[':store'] = process.env[environment + '_SHOPIFY_STORE'];
+} else {
+  var localConfig = yaml.safeLoad(fs.readFileSync('./config.yml', 'utf8'));
+}
+```
+
+We need `IS_CI` variable to determine that we are deploying via CI and need to read data from environment variables rather then from a `config.yml` file, which we need for development
```

Comment:
The `IS_CI` variable determines that we are deploying via CI, in which case we read data from the environment variables rather then from a local `config.yml` file, which is used for manual deployment.